### PR TITLE
DOCS-2790 Initial reading for clarification and minor edits

### PR DESCRIPTION
**Pull request by okilker**

### CODE REVIEW

File: docs-content/installing.html.md.erb [hunk 1]
```diff
@@ -7,14 +7,12 @@ This topic describes how to install and configure AppDynamics Platform Metrics i
...
+AppDynamics Nozzle is a Cloud Foundry (CF) component dedicated to aggregating and forwarding logs and metrics from the **Loggregator [Firehose](https://docs.cloudfoundry.org/loggregator/architecture.html#firehose)** to the AppDynamics Controller. The Controller processes the data stream including logs, HTTP events, and container metrics from all applications running on the underlying infrastructure on PCF. You can view them in the Metric Browser corresponding to the Nozzle application.
 
-When you successfully configure AppDynamics Nozzle on PCF, a default application named appd-nozzle, is created on the AppDynamics Controller. Additionally, a default tier named appd-nozzle-tier and a default node named appd-nozzle-node are created under which corresponding metrics are recorded.
+When you successfully configure AppDynamics Nozzle on PCF, a default application named **appd-nozzle**, is created on the AppDynamics Controller. Additionally, a default tier named **appd-nozzle-tier** and a default node named **appd-nozzle-node**  are created under which corresponding metrics are recorded.
 
 ## <a id='before-you-start'></a> Before You Start
```
Comment (okilker) [inline docs-content/installing.html.md.erb]: In the Before You Start section, we say:
Acquire at least one Go SDK Agent License.
I am unclear if this license has to be acquired through AppDynamics, Go/Google, or Pivotal Cloud Foundry. Can we specify, maybe with a link, on how to proceed?

File: docs-content/installing.html.md.erb [hunk 2]
```diff
@@ -29,16 +27,22 @@ Do the following:
...
+    * **Doppler Logging Endpoint**: The doppler firehose endpoint address.
+    * **Nozzle Username**: Pivotal Cloud Foundry user name.
+    * **Nozzle Password**: Pivotal Cloud Foundry user's password.
+    * **Controller Account Name**: The name of the account in the AppDynamics Controller.
+    * **Account Access Key**: The access key for the Controller account.
+    * **Controller Host**: AppDynamics host, such as https://example.cloud.appdynamics.com:8092.
 
+1. Click **Save**.
 
     * Run the  cf command to `cf curl /v2/info`. The UAA token_endpoint and Doppler Traffic Controller URL are the values of `token_endpoint` and `doppler_logging_endpoint` respectively
```
Comment (okilker) [inline docs-content/installing.html.md.erb]: In this section, I am unclear as to if this is: 

- a second set of steps to do using the Cloud Foundry command line after configuring the Ops Manager via the UI? 
- or an alternate way to perform the same configuration steps using the cf CLI. 

If it is performing the same steps, we should say: Using the Ops Manager to Configure AppDynamics Nozzle and Using the cf CLI to Configure the AppDynamics Nozzle.

File: docs-content/installing.html.md.erb (hunk 1 above)
Comment (pavankrish123) [reply docs-content/installing.html.md.erb]: It is an AppDynamics License. Perhaps we should rephrase it as AppDynamics Go SDK Agent License.

File: docs-content/installing.html.md.erb (hunk 2 above)
Comment (pavankrish123) [reply docs-content/installing.html.md.erb]: This is second set of steps to figure out the values on the form fields. This needs to be done in addition to OpsManager step

File: docs-content/using.html.md.erb [hunk 3]
```diff
@@ -72,33 +56,57 @@ The table below summarizes the environment variables, their default and allowed
...
+      <td>NOZZLE_PASSWORD</td>
+      <td>Password for the above account</td>
+      <td>none</td>
       <td>none</td>
     </tr>
     <tr>
-      <td>NOZZLE_TRAFFIC_CONTROLLER_URL</td>
-      <td>Doppler end point URL</td>
-      <td>Run cf curl /v2/info and record doppler api url</td>
+    <td>NOZZLE_TRAFFIC_CONTROLLER_URL</td>
```
Comment (okilker) [inline docs-content/using.html.md.erb]: For this Environment variable is says **NOZZLE_TRAFFIC_CONTROLLER_URL**, in the User Interface of the Settings tab, select AppDynamics Nozzle Configuration it asks for a Doppler Logging Endpoint. In the cf command it is also listed as `doppler-logging-endpoint`. 
For this page, and the Installation and Configuring page, I'm unclear as to why we have these two different terms?

File: docs-content/installing.html.md.erb [hunk 4]
```diff
@@ -3,76 +3,65 @@ title: Installing and Configuring AppDynamics Platform Monitoring
...
    A Settings tab is displayed.
 
 1. In the **Settings** tab, select **AppDynamics Nozzle Configuration**.
 ![Settings tab for AppDynamics Nozzle Configuration](./images/appd-nozzle-config.png)
 
-1. Specify the AppDynamics Nozzle configuration settings. All fields are mandatory. You cannot add multiple Nozzle configurations.
+1. Specify the AppDynamics Nozzle configuration settings. All fields are mandatory. You cannot add multiple Nozzle configurations. Click **Save**.
+
+* **UAA Token Endpoint**: The User Account and Authentication Service (UAA) API authentication, `token_endpoint` URL value.
+* **Doppler Logging Endpoint**: The doppler firehose endpoint address, `doppler_logging_endpoint` value.
```
Comment (okilker) [inline docs-content/installing.html.md.erb]: For this Environment variable it says in the User Interface of the Settings tab, select AppDynamics Nozzle Configuration it asks for a Doppler Logging Endpoint. In the cf command it is also listed as `doppler-logging-endpoint`. 
For this page, and the Overriding Configuration page it also mentioned a Doppler Traffic Controller URL, I'm unclear as to why we have these two different terms?

**Comment by jknostman3** [comment]
@okilker Thanks!

**Pull request merged**